test/Microsoft.ML.Functional.Tests/Microsoft.ML.Functional.Tests.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.ML.Functional.Tests</AssemblyName>
```

Comment:
There is no need to set `AssemblyName` in new .csproj files. This value gets defaulted to the file's name. You can remove this line.